dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "composer"
+    directory: "/"
+    ignore:
+      - dependency-name: "symfony/*"
+        update-types: ["version-update:semver-major"]
```

Comment:
Is there a way to configure this globally for every SF project? This config will have to be added to each app now? Which might be a good idea.

Reply:
This will need to be rolled out to all symfony repo's. 

I couldn't find a way to handle it globally. This has been added to the backlog.

Reply:
@johanib Have you verified that this actually works? Because I've been struggling with this same issue and nothing seems to work...

https://github.com/simplesamlphp/simplesamlphp/blob/master/.github/dependabot.yml#L18-L27

There are many bug reports in the dependabot-repo related to this, but they don't seem to pick this up:
https://github.com/dependabot/dependabot-core/issues/10122